magento/inventory
# Replace IsProductSalable and IsProductSalableForRequestedQty usages.

Replace IsProductSalable and IsProductSalableForRequestedQty usages with new AreProductsSalable and AreProductsSalableForRequestedQty services.





### Description (*)


### Fixed Issues (if relevant)

1. none.

### Manual testing scenarios (*)

1. Create all types of products on default and custom stocks.
2. Verify products visibility on storefront.
4. Create order, invoice, shipment and credit memo with all product types.
5. Verify, no errors occurred.

### Questions or comments


### Contribution checklist (*)
